src/main/webapp/index.html

```diff
@@ -4,27 +4,45 @@
     <meta charset="UTF-8">
     <title>Student Recommendations</title>
     <meta name="viewport" content="width=device-width, initial-scale=1">
+    <script src="/scripts/mapscript.js"></script>
+    <script type='text/javascript' src='/scripts/config.js'></script>
+    <script
+      src="https://maps.googleapis.com/maps/api/js?key=AIzaSyDME967qxzDl6dYJTT_d25n6yPHM1LOehw&callback=initMap&libraries=&v=weekly"
```

Comment:
Should we hide the API key?

Reply:
After reviewing this [https://developers.google.com/maps/documentation/javascript/get-api-key#key-restrictions](url) resource, it seems like the API key doesn't need to be hidden, as long as it is restricted.

Reply:
unfortunately, I can't check bc the page is 404 for me :(